sim/warrior/shouts.go

```diff
@@ -33,16 +33,17 @@ func (warrior *Warrior) makeShoutSpellHelper(actionID core.ActionID) *core.Spell
 
 		ApplyEffects: func(sim *core.Simulation, _ *core.Unit, _ *core.Spell) {
 			// Actual shout effects are handled in core/buffs.go
-			warrior.shoutExpiresAt = sim.CurrentTime + warrior.shoutDuration
+			warrior.shoutExpiresAt = sim.CurrentTime + warrior.shoutDuration + extraDuration
 		},
 	})
 }
 
 func (warrior *Warrior) makeShoutSpell() *core.Spell {
 	if warrior.ShoutType == proto.WarriorShout_WarriorShoutBattle {
-		return warrior.makeShoutSpellHelper(core.ActionID{SpellID: 47436})
+		return warrior.makeShoutSpellHelper(core.ActionID{SpellID: 47436}, 0)
```

Comment:
Same comment from previous PR: Where is Glyph of Battle handled if not here?

Reply:
I don't think its handled, I'll check and add it here if not.